apps/web/src/app/tournament/create/_components/addWishDialog/AddByLinkDialog.tsx

```
@@ -59,10 +58,10 @@ function AddByLinkDialog({ open, onOpenChange, onSubmit }: AddByLinkDialogProps)
             label="링크 URL"
             placeholder="복사한 링크를 입력해주세요."
             value={url}
-            onChange={e => handleChange(e.target.value)}
+            onChange={handleUrlChange}
             left={<LinkIconFill className="size-5" />}
             aria-invalid={hasError}
-            {...(hasError ? { helperText: '올바른 URL 형식으로 입력해주세요.' } : {})}
+            helperText={hasError ? '올바른 URL 형식으로 입력해주세요.' : undefined}
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)